wf/entrypoint.py

```diff
@@ -0,0 +1,430 @@
+    qiime_ref_taxonomy: Optional[QiimeRefTaxonomy],
+    qiime_ref_tax_custom: Optional[LatchFile],
+    classifier: Optional[LatchFile],
+    kraken2_ref_taxonomy: Optional[Kraken2RefTaxonomy],
+    kraken2_ref_tax_custom: Optional[LatchFile],
+    kraken2_assign_taxlevels: Optional[str],
+    sintax_ref_taxonomy: Optional[SintaxRefTaxonomy],
+    addsh: bool,
+    sidle_ref_taxonomy: Optional[SidleRefTaxonomy],
+    sidle_ref_tax_custom: Optional[List[LatchFile]],
+    sidle_ref_tree_custom: Optional[LatchFile],
+    metadata_category: Optional[str],
+    metadata_category_barplot: Optional[str],
+    qiime_adonis_formula: Optional[str],
+    picrust: bool,
+    sbdiexport: bool,
+    ancom: bool,
+    ancombc: bool,
+    ancombc_formula: Optional[str],
+    ancombc_formula_reflvl: Optional[str],
+    report_abstract: Optional[str],
+    skip_fastqc: bool,
+    skip_cutadapt: bool,
+    skip_dada_quality: bool,
+    skip_barrnap: bool,
+    skip_qiime: bool,
+    skip_qiime_downstream: bool,
+    skip_taxonomy: bool,
+    skip_dada_taxonomy: bool,
+    skip_dada_addspecies: bool,
+    skip_barplot: bool,
+    skip_abundance_tables: bool,
+    skip_alpha_rarefaction: bool,
+    skip_diversity_indices: bool,
+    skip_multiqc: bool,
+    skip_report: bool,
+    multiqc_methods_description: Optional[str],
+    report_template: Optional[str],
+    report_css: Optional[str],
+    report_logo: Optional[str],
+    extension: Optional[str] = "/*_R{1,2}_001.fastq.gz",
+    min_read_counts: Optional[int] = 1,
+    cutadapt_min_overlap: Optional[int] = 3,
+    cutadapt_max_error_rate: Optional[float] = 0.1,
+    trunc_qmin: Optional[int] = 25,
+    trunc_rmin: Optional[float] = 0.75,
+    max_ee: Optional[int] = 2,
+    min_len: Optional[int] = 50,
+    sample_inference: Optional[str] = "independent",
+    vsearch_cluster_id: Optional[float] = 0.97,
+    orf_start: Optional[int] = 1,
+    stop_codons: Optional[str] = "TAA,TAG",
+    dada_ref_taxonomy: Optional[DADATaxonomy] = DADATaxonomy.silva_138,
+    pplace_alnmethod: Optional[str] = "hmmer",
+    kraken2_confidence: Optional[float] = 0.0,
+    cut_its: Optional[str] = "none",
+    its_partial: Optional[int] = 0,
+    exclude_taxa: Optional[str] = "mitochondria,chloroplast",
+    min_frequency: Optional[int] = 1,
+    min_samples: Optional[int] = 1,
+    diversity_rarefaction_depth: Optional[int] = 500,
+    tax_agglom_min: Optional[int] = 2,
+    tax_agglom_max: Optional[int] = 6,
+    ancom_sample_min_count: Optional[int] = 1,
+    ancombc_effect_size: Optional[float] = 1.0,
+    ancombc_significance: Optional[float] = 0.05,
+    report_title: Optional[str] = "Summary of analysis results",
+    seed: Optional[int] = 100,
+    max_cpus: Optional[int] = 16,
+    max_memory: Optional[str] = "128.GB",
+    max_time: Optional[str] = "240.h",
+    outdir: LatchOutputDir = LatchOutputDir("latch:///Ampliseq"),
+) -> None:
+    shared_dir = Path("/nf-workdir")
+    rename_current_execution(str(run_name))
```

Comment:
maybe do this in the previous task? the earlier the better from a UX perspective?